<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetAudio</class>
 <widget class="QWidget" name="WidgetAudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="txtPrefixAndName">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRenameAudio">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Audio States:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbStates">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="btnAddState">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveState">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnRenameState">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnOrderStateBack">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnOrderStateForward">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpState">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Audio State Info</string>
     </property>
     <layout class="QVBoxLayout" name="grpStateLayout"/>
    </widget>
   </item>
  </layout>
  <action name="actionRenameState">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-rename.png</normaloff>:/icons16x16/generic-rename.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rename State</string>
   </property>
   <property name="toolTip">
    <string>Rename state</string>
   </property>
  </action>
  <action name="actionAddState">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add State</string>
   </property>
   <property name="toolTip">
    <string>Add a new state for this audio instance</string>
   </property>
  </action>
  <action name="actionRemoveState">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remo&amp;ve State</string>
   </property>
   <property name="toolTip">
    <string>Remove the currently selected state</string>
   </property>
  </action>
  <action name="actionOrderStateBackwards">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-left.png</normaloff>:/icons16x16/generic-left.png</iconset>
   </property>
   <property name="text">
    <string>Order State &amp;Backwards</string>
   </property>
   <property name="toolTip">
    <string>Shift the index of the current state back by 1</string>
   </property>
  </action>
  <action name="actionOrderStateForwards">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-right.png</normaloff>:/icons16x16/generic-right.png</iconset>
   </property>
   <property name="text">
    <string>Order State For&amp;wards</string>
   </property>
   <property name="toolTip">
    <string>Shift the index of the current state forward by 1</string>
   </property>
  </action>
  <action name="actionImportWaves">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>Import &amp;Waves</string>
   </property>
   <property name="toolTip">
    <string>Import the selected waves from the Audio Manager</string>
   </property>
  </action>
  <action name="actionRemoveWave">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Wave</string>
   </property>
   <property name="toolTip">
    <string>Remove the selected wave</string>
   </property>
   <property name="shortcut">
    <string>Del, Backspace</string>
   </property>
  </action>
  <action name="actionOrderWaveUpwards">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-up.png</normaloff>:/icons16x16/generic-up.png</iconset>
   </property>
   <property name="text">
    <string>Order Wave &amp;Upwards</string>
   </property>
   <property name="toolTip">
    <string>Shift the selected wave to be ordered earlier in the sequence</string>
   </property>
  </action>
  <action name="actionOrderWaveDownwards">
   <property name="icon">
    <iconset resource="res/DesignerTool.qrc">
     <normaloff>:/icons16x16/generic-down.png</normaloff>:/icons16x16/generic-down.png</iconset>
   </property>
   <property name="text">
    <string>Order Wave &amp;Downwards</string>
   </property>
   <property name="toolTip">
    <string>Shift the selected wave to be ordered later in the sequence</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res/DesignerTool.qrc"/>
 </resources>
 <connections/>
</ui>
